multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/auditlogging/FilesApiServiceAuditLog.java

```diff
@@ -0,0 +1,104 @@
+                                                                                    performedAction,
+                                                                                    Messages.FILE_UPLOAD_FROM_URL_AUDIT_LOG_CONFIG,
+                                                                                    createAuditLogStartUploadFromUrlConfigurationIdentifier(fileUrl)),
+                                                          ConfigurationChangeActions.CONFIGURATION_CREATE);
+    }
+
+    public void logGetUploadFromUrlJob(String username, String spaceGuid, String namespace, String jobId) {
+        String performedAction = MessageFormat.format(Messages.GET_INFO_FOR_UPLOAD_URL_JOB_AUDIT_LOG_MESSAGE, spaceGuid);
+        auditLoggingFacade.logDataAccessAuditLog(new AuditLogConfiguration(username,
+                                                                           spaceGuid,
+                                                                           performedAction,
+                                                                           Messages.UPLOAD_FROM_URL_JOB_INFO_AUDIT_LOG_CONFIG,
+                                                                           createAuditLogGetUploadFromUrlJobConfigurationIdentifier(namespace,
+                                                                                                                                    jobId)));
+    }
+
+    private Map<String, String> createFileMetadataConfigurationIdentifier(FileMetadata fileMetadata) {
+        Map<String, String> identifiers = new HashMap<>();
+
+        identifiers.put(FILE_ID_PROPERTY_NAME, fileMetadata.getId());
+        identifiers.put(DIGEST_PROPERTY_NAME, fileMetadata.getDigest());
+        identifiers.put(DIGEST_ALGORITHM_PROPERTY_NAME, fileMetadata.getDigestAlgorithm());
+        identifiers.put(SIZE_PROPERTY_NAME, Objects.toString(fileMetadata.getSize()));
+        identifiers.put(NAMESPACE_PROPERTY_NAME, fileMetadata.getNamespace());
+
+        return identifiers;
+    }
+
+    private Map<String, String> createAuditLogGetFilesConfigurationIdentifier(String namespace) {
+        Map<String, String> identifiers = new HashMap<>();
+
+        identifiers.put(NAMESPACE_PROPERTY_NAME, namespace);
+
+        return identifiers;
+    }
+
+    private Map<String, String> createAuditLogGetUploadFromUrlJobConfigurationIdentifier(String namespace, String jobId) {
+        Map<String, String> identifiers = new HashMap<>();
+
+        identifiers.put(NAMESPACE_PROPERTY_NAME, namespace);
+        identifiers.put(JOB_ID_PROPERTY_NAME, jobId);
+
+        return identifiers;
+    }
+
+    private Map<String, String> createAuditLogStartUploadFromUrlConfigurationIdentifier(String fileUrl) {
+        Map<String, String> identifiers = new HashMap<>();
+
+        identifiers.put(FILE_URL_PROPERTY_NAME, fileUrl);
+
+        return identifiers;
+    }
```

Comment:
you can use Map.of for shorted syntax

Reply:
initially it was with Map.of but the problem is that some values are null and map.of request not null values